posthog/api/action.py

```diff
@@ -304,28 +294,56 @@ def trends(self, request: request.Request, *args: Any, **kwargs: Any) -> Respons
         actions = actions.filter(deleted=False)
         actions_list = []
 
-        parsed_actions = self._parse_actions(request)
+        parsed_actions = self._parse_entities('actions')
+        parsed_events = self._parse_entities('events')
+        def _trend_entity_factory(id, name):
+            return {
+                'action': {
+                    'id': id,
+                    'name': name
+                },
+                'label': name,
+                'count': 0,
+                'breakdown': []
+            }
 
+        def _process_trend_entity(entity, filters={}, entity_type=None):
+            if entity_type == 'action':
```

Comment:
not a huge fan of magic strings here for type comparison. using this for a moment because of inconsistency between an actual action type vs a dict that has an event id